src/App.js

```diff
@@ -9,16 +10,18 @@ import {
   UALProviderSwitch,
   WalletProvider
 } from "./context/walletProvider/walletProviderFacade";
+import { ethBasedChainId } from "./config";
 
 const HomePage = lazy(() => import("./features/home"));
 const EthPage = lazy(() => import("./features/eth"));
+const LockPage = lazy(() => import("./features/Lock/lock"));
 
 function App() {
   return (
     <ErrorBoundary fallback={<Error />}>
       <Suspense fallback={<Loading />}>
         <UseWalletProvider
-          chainId={1} // 5 GOERLI
+          chainId={Number(ethBasedChainId)} // 5 GOERLI // TODO: make it flexible (137 for Matic)
```

Comment:
this is not gonna solve the "make it flexible"

In our use case, we need to use 137 for the locking (Matic) but for the bridge we need to use 1 (ethereum). System needs to be flexible for that.

Right now if I place 137 here I am able to lock, but I won't be able to use the bridge

Reply:
@kesar yo right. I'm on that right now